src/Backend/Synapse/SynapseImportOptions.php

```diff
@@ -11,29 +11,40 @@ class SynapseImportOptions extends ImportOptions
     public const CREDENTIALS_MANAGED_IDENTITY = 'MANAGED_IDENTITY';
     public const CREDENTIALS_SAS = 'SAS';
 
+    /** @deprecated staging table is created by user */
     public const TEMP_TABLE_HEAP = 'HEAP';
+    /** @deprecated staging table is created by user */
     public const TEMP_TABLE_HEAP_4000 = 'HEAP4000';
+    /** @deprecated staging table is created by user */
     public const TEMP_TABLE_COLUMNSTORE = 'COLUMNSTORE';
+    /** @deprecated staging table is created by user */
     public const TEMP_TABLE_CLUSTERED_INDEX = 'CLUSTERED_INDEX';
 
     public const DEDUP_TYPE_CTAS = 'CTAS';
+    /** @deprecated use DEDUP_TYPE_CTAS */
     public const DEDUP_TYPE_TMP_TABLE = 'TMP_TABLE';
 
     public const TABLE_TYPES_CAST = 'TABLE_HAS_TYPES_DEFINED';
     public const TABLE_TYPES_PRESERVE = 'TABLE_TYPES_PRESERVE';
 
-    /** @var string */
-    private $importCredentialsType;
+    public const SAME_TABLES_REQUIRED = true;
+    public const SAME_TABLES_NOT_REQUIRED = false;
```

Comment:
nazval bych to `SAME_TABLES_DEFAULT_OPTION` kdyz jedinej vyznam tady teto konstanty je reprezentovat default tady v constructoru. A dal bych ji private

Reply:
No možu tam přidat `SAME_TABLES_DEFAULT_OPTION=SAME_TABLES_NOT_REQUIRED` ale moc mě to smysl nedává protože netušíš co to reprezentuje ta konstatna. Private nedává smysl už vůbec, protože pak to nemožeš nainstancovat s NOT_REQUIRED, nikde přeca není napsané, že když chceš default tak ten poslední parametr máš přeskočit, budeš chtět třeba `$isTypedTable ? SAME_TABLES_REQUIRED : SAME_TABLES_NOT_REQUIRED`

Reply:
jo, ja pocital s tim, ze kdyz to nekdo bude chtit pouzit, tak si tam das true/false. ale takto jak to vysvetlujes tak mi to dava smysl

Reply:
Nedá sa použít true/false musíš použít konstantu jinak ti neprojde phpstan :)